spring-cloud-dataflow-composed-task-runner/src/test/java/org/springframework/cloud/dataflow/composedtaskrunner/ComposedTaskRunnerConfigurationNoPropertiesTests.java

```diff
@@ -83,6 +83,6 @@ public void testComposedConfiguration() throws Exception {
 		assertThat(composedTaskProperties.getTransactionIsolationLevel()).isEqualTo("ISOLATION_REPEATABLE_READ");
 
 		Assert.notNull(job.getJobParametersIncrementer(), "JobParametersIncrementer must not be null.");
-		verify(taskOperations).launch("AAA", new HashMap<>(0), new ArrayList<>(0));
+		verify(taskOperations).launch("AAA", new HashMap<>(0), Collections.singletonList("--spring.cloud.task.parent-execution-id=1"));
```

Comment:
Do you mind adding a test (any location is fine w/ me) that verifies the case when it comes up null?